nucliadb_vectors/src/data_point/ops_hnsw.rs

```diff
@@ -140,11 +141,14 @@ impl<'a, DR: DataRetriever> HnswOps<'a, DR> {
         // best solution. This algorithm takes a lazy approach to computing the similarity of
         // candidates.
 
+        const MAX_VECTORS_TO_PRELOAD: u32 = 20_000;
```

Comment:
nit: I know you have tested it thoroughly, but is it worth having an env var to tweak this if needed? I'm ok with not having it too - just a thought

Reply:
Passing the settings down to here would be a massive pain, so I'd rather not :sweat_smile:. I don't think we have a good way to test which value is best in production anyway, there is too much variance from search to search. I've only been able to do some experiments by working with a fixed dataset and set of queries (seeded rng), so I don't see how we would see in production that we want to tweak this.